src/test/resources/3_0/security.yaml

```diff
@@ -45,7 +45,171 @@ paths:
       responses:
         '200':
           description: OK
+  /token:
```

Comment:
Please delete this route and all component schemas that use it. The auth client should take card of handling this. I understand that your local server is vending this. The component security scheme already defines what is needed here.

Reply:
Fixed. Removed from chema.